charts/opentelemetry-demo/templates/_helpers.tpl

```diff
@@ -57,3 +57,30 @@ app.kubernetes.io/component: {{ .name}}
 {{- $mergedEnvs = concat $mergedEnvs $envOverrides }}
 {{- mustToJson $mergedEnvs }}
 {{- end }}
+
+{{/*
+Return the appropriate apiVersion for ingress.
+*/}}
+{{- define "ingress.apiVersion" -}}
+  {{- if and (.Capabilities.APIVersions.Has "networking.k8s.io/v1") (semverCompare ">= 1.19-0" .Capabilities.KubeVersion.Version) -}}
```

Comment:
Is this to support old versions of k8s?  I'd rather the chart target only versions of k8s that the k8s team are still supporting.

Reply:
yeah, the code was copied from the Collector chart.  I'm happy to simplify this so we only support recent versions of K8s.  What would be the min version we are targeting for support? 1.23?

Reply:
Dang collector chart, the maintainers of that thing should really up their game lol

I believe 1.23 is the minimum supported version ya.